src/browser/stacktrace/utils.ts

```diff
@@ -132,13 +205,29 @@ export class ShallowStackFrames {
         this._framesMap.delete(key);
     }
 
-    isNested(childFrames: RawStackFrames): boolean {
+    private _getParentStackFrame(childFrames: RawStackFrames): RawStackFrames | null {
         for (const parentFrames of this._framesMap.values()) {
             if (childFrames.length !== parentFrames.length && childFrames.endsWith(parentFrames)) {
-                return true;
+                return parentFrames;
             }
         }
 
-        return false;
+        return null;
+    }
+
+    areInternal(childFrames: RawStackFrames): boolean {
+        const parentStackFrame = this._getParentStackFrame(childFrames);
+
+        if (!parentStackFrame) {
+            return false;
+        }
+
+        const isNodeModulesFrame = (frame: string): boolean => frame.includes("/node_modules/");
+        const isNodeInternalFrame = (frame: string): boolean => frame.includes(" (node:");
```

Comment:
Here it is too expensive to parse frames, because "areInternal" method will be called a lot of times on every browser command. So we are checking node_modules frames and internal frames by these functions